testgrid/config.yaml

```diff
@@ -656,6 +656,14 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-ibmcloud-test
 - name: pull-cluster-api-provider-ibmcloud-check
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-ibmcloud-check
+- name: pull-cluster-api-provider-vsphere-verify-fmt
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-vsphere-verify-fmt
+- name: pull-cluster-api-provider-vsphere-verify-lint
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-vsphere-verify-lint
+- name: pull-cluster-api-provider-vsphere-verify-vet
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-vsphere-verify-vet
+- name: pull-cluster-api-provider-vsphere-verify-markdown
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-vsphere-verify-markdown
```

Comment:
FWIW I am pretty sure you don't need these; you can find out by removing and seeing if our tests yell at you

Reply:
I'm more than happy to test this theory :)

Reply:
Alright @spiffxp, seems good to go. Thanks for finding this!

Reply:
Sorry I was AFK for a bit, but this goes back to me asking about this in #sig-testing [earlier today](https://kubernetes.slack.com/archives/C09QZ4DQB/p1562867544339400). I am going to clean up all the vSphere tests to use annotations and get it out of `testgrid.yaml`, but I'm waiting until the mass cleanup to delete the gcs buckets definitions like this. =)